test/test_pcan.py

```diff
@@ -34,6 +37,17 @@ def tearDown(self) -> None:
             self.bus.shutdown()
             self.bus = None
 
+    def _mockGetValue(self, Channel, Parameter):
+        """
+        This method is used as mock for GetValue method of PCANBasic object.
+        Only a subset of parameters are supported.
+        """
+        if Parameter == PCAN_API_VERSION:
+            return PCAN_ERROR_OK, self.PCAN_API_VERSION_SIM.encode("ascii")
+        raise NotImplementedError(
+            f"No mock return value specified for parameter {Parameter}"
+        )
```

Comment:
It's convention to use lowercase variables. :)

```suggestion
    def _mockGetValue(self, channel, parameter):
        """
        This method is used as mock for GetValue method of PCANBasic object.
        Only a subset of parameters are supported.
        """
        if parameter == PCAN_API_VERSION:
            return PCAN_ERROR_OK, self.PCAN_API_VERSION_SIM.encode("ascii")
        raise NotImplementedError(
            f"No mock return value specified for parameter {parameter}"
        )
```